tac/agents/v1/base/reactions.py

```diff
@@ -320,7 +320,8 @@ def _register_to_tac(self, controller_pbk: str) -> None:
         self.game_instance.controller_pbk = controller_pbk
         self.game_instance._game_phase = GamePhase.GAME_SETUP
         msg = Register(self.crypto.public_key, self.crypto, self.agent_name).serialize()
-        self.out_box.out_queue.put(OutContainer(message=msg, message_id=0, dialogue_id=0, destination=controller_pbk))
+        self.mailbox.outbox.put(ByteMessage(to=self.game_instance.controller_pbk, sender=self.crypto.public_key,
```

Comment:
controller_pbk is created as local var above; remove either one of both

Reply:
where, exactly? `controller_pbk` is the argument of the `_register_to_tac` function.

Reply:
yes, just wondering why you first assign it and then call it for `to=`, could just do `to=controller_pbk`?...nothing major though

Reply:
right, will fix